### PR TITLE
Feature/69 add open url option

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added OpenUrl option for vault account class
- Updated readme file with the new changes in the vda-account vda-login

### CODE REVIEW

File: components/account/dev/serve.vue [hunk 1]
```diff
@@ -7,9 +7,10 @@ export default defineComponent({
   name: "ServeDev",
   data() {
     return {
-      contextName: "Verida: New Account Component",
+      contextName: "Verida: Nkew Account Component",
```
Comment (dmty) [inline components/account/dev/serve.vue:10]: 🤔

File: components/account/README.md [hunk 2]
```diff
@@ -62,11 +63,13 @@ export default defineComponent({
     return {
       contextName: "Verida: Account Component",
       logo: "https://assets.verida.io/verida_login_request_logo_170x170.png",
+      openUrl: "http://172.20.10.3:8081/",
     };
   },
   methods: {
     onSuccess(response: any) {
-      this.$vdaClient.initUser(response);
```
Comment (dmty) [inline components/account/README.md:69]: Is the initUser not suggested anymore?

**Review by dmty — APPROVED** [review]
Don't see anything blocking

File: components/account/dev/serve.vue (hunk 1 above)
Comment (cmcWebCode40) [reply components/account/dev/serve.vue:10]: Ah, typo error

File: components/account/README.md (hunk 2 above)
Comment (cmcWebCode40) [reply components/account/README.md:69]: Yes, a refactoring was done to remove that, that's was when we were using vuex to manage state and also exposing the  verida helper function .

**Pull request merged**